scala/README.md

```diff
@@ -129,7 +130,17 @@ Please read the [Scala Style Guide] carefully. The main points to consider are:
 
 0. No double blank lines, anywhere.
 
-0. No trailing whitespace at the end of lines, they cause problems when diffing between files or between versions. Configure your text editor to do this for you.
+0. No trailing whitespace at the end of lines, they cause problems when diffing between files or between versions. Configure your text editor to do this for you automatically.
+
+0. In general, obey English rules and mathematical conventions for punctuation:
+
+    1. A single space after and no space before `,`, `:`, `;`, `)`, etc.
+
+    1. A single space before and no space after `(`, except for method invocation or declaration.
```

Comment:
this is a bit ambiguous since you specify both before and after. maybe rephrase to: 

> a single space before `(`, except for method invocation and declaration. never a space after.